docs/guides/napari_models.md

```diff
@@ -20,14 +20,30 @@ The three main components:
 
 The separation of the python models from viewer GUI code allows:
 
-* the python model to be easily run headless (without opening the napari GUI
-  interface), for example when performing batch analysis
 * analysis plugins to be developed without worrying about the GUI
   aspect
 * napari to have the option to move away from the rendering backend currently
   used
 * tests to be easily run headlessly
 
+There is also intention to allow the python model to be run headlessly (without
+opening a napari GUI interface), which may be useful for example when performing
+batch analysis.
+
+Headless mode can be achieved by not showing the viewer:
+
+```python
+viewer = napari.Viewer(show=False)
+# or
+viewer_model = napari.components.ViewerModel()
+```
+
+Note that currently using `napari.Viewer(show=False)` will *not* prevent
+Qt to be imported *unless* QtPy or any of the Qt components is uninstalled.
```

Comment:
The possible workaround for the problem of crashing the napari application on creating QAppliation is to set `QT_QPA_PLATFORM=offscreen` in the environment. 


![obraz](https://user-images.githubusercontent.com/3826210/185050890-ba8e3860-346e-4b60-99e9-aa488f0a66c9.png)

I do not know if this will allow for proper render, for example, screenshots, but it should allow solving problems like #3686 or just make it possible to use on a server, the napari installation from conda.

Reply:
Hmm I am not sure this would be the best place for this information, though admittedly there are no docs on how to run napari headless...

Reply:
Also here you add information that import Qt could be problematic without explanation why.

Reply:
Good point, I was mostly trying to summarise the info here: https://github.com/lucyleeow/napari/pull/7#discussion_r946160065

Note that I don't have a deep understanding of this. My guess would be if you are running in a CLI Qt may not work and crash?

Reply:
The first command on sreenshot is the try of run napari (create viewer) on serwer without X. On try to create QApplication, if qt cannot connect to display then it print messages from terminal and abort application. And it will happen always on create viewer if qt libs are installed and default plugin is selected.

The environment variable told Qt backend that rendering should be done offscreen. So QApplication creating will end with success.

Reply:
Thank you that makes sense. I think I will add a section to the docs about tips on running napari headless as this info may be useful to users, whereas the rest of this doc is more aimed at people interested in contributing to napari. Also it doesn't seem to fit with the content of these docs. Thanks